junit5/src/main/java/cz/xtf/junit5/extensions/OpenShiftRecorderService.java

```diff
@@ -534,7 +535,7 @@ private String dirNameForTest(ExtensionContext context) {
         if (context.getTestMethod().isPresent()) {
             return context.getTestClass().get().getName() + "." + context.getDisplayName();
         } else {
-            return context.getTestClass().get().getName();
+            return context.getTestClass().get().getName() + "-" + UUID.randomUUID();
```

Comment:
The whole uuid seems long, can it be shortened to just few letters?

Reply:
I can't guess what's the use for generating a UUID and then shortening it, i.e. losing its format, one of the ingredients that make the recipe unique. Hence I went for `ThreadLocalRandom.current().nextInt()`.

@mnovak1 let me know whether this works for you.

Reply:
Nice, I wasn't even aware of `ThreadLocalRandom` class. Reading JavaDoc it seems to be good for the job. Only problem might be with negative values which does not look good. However `ThreadLocalRandom.nextInt(int bound)` seems to return only positive numbers based on JavaDoc. Can it be used?

Reply:
Right, good catch! Fixed.